src/javascripts/ng-admin/es6/lib/View/View.js

```diff
@@ -3,6 +3,7 @@ import Entry from "../Entry";
 class View {
     constructor(name) {
         this.entity = null;
+        this._dataStore = null;
```

Comment:
Same remark as for `Entity`: `View` instances should be immutable and not contain a `DataStore`. I think there should only be one single `DataStore` instance, probably linked to by the `Application` instance.